wp-content/themes/pub/wporg-learn-2020/css/vendor/_overrides.scss

```diff
@@ -259,7 +259,6 @@ p {
 
 .lp-details {
 	font-size: 14px;
-	overflow: hidden;
```

Comment:
This is hiding the outline style for the actions. I haven't been able to find a reason to keep it.